sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -3204,12 +3284,12 @@
                     "calculateServer": false,
                     "allowCalculateOverride": false,
                     "validate": {
-                        "required": true,
+                        "required": false,
                         "onlyAvailableItems": false,
-                        "customMessage": "An entrance requirement is required.",
+                        "customMessage": "",
                         "minSelectedCount": "",
                         "maxSelectedCount": "",
-                        "custom": "",
+                        "custom": "var shouldSelectOneOfTheAboveOrAnotherOption =\r\n   ( ( data.entranceRequirements.minHighSchool ||\r\n       data.entranceRequirements.hasMinimumAge ||\r\n\t   data.entranceRequirements.requirementsByInstitution ||\r\n\t   data.entranceRequirements.requirementsByBCITA ) && \r\n       data.entranceRequirements.noneOfTheAboveEntranceRequirements );\r\nconsole.log(\"shouldSelectOneOfTheAboveOrAnotherOption \"+shouldSelectOneOfTheAboveOrAnotherOption);\r\n\r\nvar shouldSelectAtLeastOneOption = \r\n\t   ( !data.entranceRequirements.minHighSchool &&\r\n         !data.entranceRequirements.hasMinimumAge &&\r\n\t     !data.entranceRequirements.requirementsByInstitution &&\r\n\t     !data.entranceRequirements.requirementsByBCITA && \r\n         !data.entranceRequirements.noneOfTheAboveEntranceRequirements );\r\nconsole.log(\"shouldSelectAtLeastOneOption \"+shouldSelectAtLeastOneOption);\r\n\r\n\r\nvalid = shouldSelectAtLeastOneOption ? \"An entrance requirement is required.\" : shouldSelectOneOfTheAboveOrAnotherOption ? \"Please select one of the entrance requirements or 'None of the above'.\" : true;\r\n",
```

Comment:
pls, remove the console.log

Reply:
and how about `"Entrance requirement is required.` and `\"Please select the entrance requirements or 'None of the above'.`

Reply:
"An entrance requirement is required."  is how it was before.
The other one should something like: "Please select at least one of the entrance requirements or 'None of the above'." 
Maybe @hellolynn-tbtb can tell us what she prefers in this case.

Reply:
@andrepestana-aot both are good and I like @ann-aot  because it's shorter but please go ahead with "Please select at least one of the entrance requirements or 'None of the above'." - it has a lower grade reading level (surprisingly with more text)